Samples/MicrosoftSpatializerSample/Assets/packages.config

```diff
@@ -1,7 +1,7 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="Microsoft.MixedReality.Toolkit.Extensions" version="2.1.0" />
-  <package id="Microsoft.MixedReality.Toolkit.Foundation" version="2.1.0" />
-  <package id="Microsoft.MixedReality.Toolkit.Tools" version="2.1.0" />
-  <package id="Microsoft.SpatialAudio.Spatializer.Unity" version="0.1.15-alpha" />
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.MixedReality.Toolkit.Extensions" version="2.1.0" />

```

Comment:
I'm surprised that the Unity version got upgraded, but MRTK was left at 2.1. Should this be upgraded to latest?

Reply:
Hmm, good question! I think we can update to the latest MRTK but I'll do that as a separate commit in develop. I want to get this in and make progress with UPM publishing work.